### PR TITLE
first try at readthedocs

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+sphinx:
+  configuration: docs/conf.py
```
Comment (stuartarchibald) [inline .readthedocs.yml]: should this be `docs/source/conf.py`?

Comment (seibert) [reply .readthedocs.yml]: Good catch!

Comment (seibert) [reply .readthedocs.yml]: fixed

**Comment by seibert** [comment]
merging

**Pull request merged**